astropy/stats/biweight.py

```diff
@@ -362,7 +360,9 @@ def biweight_midvariance(data, c=9.0, M=None, axis=None,
     else:
         if axis is None:
             n = data.size
-        else:
+        elif type(axis) in (tuple, list):
```

Comment:
I think `isinstance(axis, (tuple, list))` may be slightly preferred. Also, not that it really matters, but for similar loops over small numbers of elements I found that `functools.reduce` is much faster for these small number of elements:
```
from functools import reduce
import operator
import numpy as np
shape = tuple(range(5))
axes = [2, 3]
%timeit reduce(operator.mul, (shape[axis] for axis in axes), 1)
# 1000000 loops, best of 5: 474 ns per loop
%timeit np.prod([shape[axis] for axis in axes])
# 100000 loops, best of 5: 8.76 µs per loop
```

Reply:
I changed to `isinstance`.  Interestingly, no one suggested this in my `numpy` PR.

I didn't implement `reduce`, but I can do that if you want (thanks for the tip!).

Reply:
I think you forgot the change to `isinstance`, but no big deal.

Reply:
Ah, I changed it in the `_expand_dims` function  (https://github.com/astropy/astropy/blob/master/astropy/stats/funcs.py#L77), but forgot to also change it here.